website/docs/d/policy_gateway_dns_forwarder.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+subcategory: "DNS"
+layout: "nsxt"
+page_title: "NSXT: policy_gateway_dns_forwarder"
+description: A policy gateway DNS forwarder data source.
+---
+
+# nsxt_policy_gateway_dns_forwarder
+
+This data source provides information about policy gateways DNS forwarder.
+
+This data source is applicable to NSX Policy Manager, NSX Global Manager and VMC.
+
+## Example Usage
+
+```hcl
+data "nsxt_policy_gateway_dns_forwarder" "my_dns_forwarder" {
+  display_name = "dns-forwarder1"
+  gateway_path = data.nsxt_policy_tier1_gateway.path
+}
+```
+
+## Argument Reference
+
+* `gateway_path` - (Required) Path for the gateway.
```

Comment:
As GW path is not required in the schema, shouldn't it be optional here?

Reply:
Updated.